experiments/AtmosLES/stable_bl_les.jl

```diff
@@ -11,11 +11,19 @@ function main()
         metavar = "prescribed|bulk"
         arg_type = String
         default = "bulk"
+
+        "--CP-Version"
+        help = "specify CLIMAParameters version"
+        arg_type = Union{String, Nothing}
+        default = nothing
     end
 
     cl_args = ClimateMachine.init(parse_clargs = true, custom_clargs = sbl_args)
 
     surface_flux = cl_args["surface_flux"]
+    if !isnothing(cl_args["CP_Version"])
+        include("clima_param_defs_$(cl_args["CP_Version"]).jl")
```

Comment:
This include must be located at the top level scope (we’re defining global methods). Alternatively, we might be able to use `eval`. Looping in @jakebolewski

Reply:
Got it. For reference, this is what that file might look like:

```
CLIMAParameters.Atmos.SubgridScale.
                C_smag(::EarthParameterSet) =
                0.30016549507297424
CLIMAParameters.Atmos.SubgridScale.
                C_drag(::EarthParameterSet) =
                0.0009143650169447696
```

Reply:
shouldn't this take a path to a file (with the method overloads?)  It seems a bit hacky to do it this way but probably will work for now.   `include()` evals in the global toplevel scope of the containing module so it should be fine I would think

Reply:
FYI, as you said this does not work. I just checked and the values used for these two parameters in the simulation are the default ones.

Reply:
Isn't that because of the way the methods are imported into the module namespace? 
https://github.com/CliMA/ClimateMachine.jl/blob/a5dd39fb7abab30523d33a0f3f3703283abcdad5/experiments/AtmosLES/stable_bl_model.jl#L42

doing it this way they are not available for extension:

https://docs.julialang.org/en/v1/manual/modules/#Summary-of-module-usage

Reply:
> `include() evals in the global toplevel scope of the containing module so it should be fine I would think

That would seem odd. Where/when exactly does the include get executed? This `include` is inside a function. I think we need to `eval` into `Main` scope.

Reply:
My thought/suggestion would be to define
```julia
    function set_clima_parameters(filename)
      eval(:(include($filename)))
    end
```
And call this in the function in `main`